settings.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
+
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+DATABASES = {
+'default': {
+    'ENGINE': 'django.db.backends.postgresql_psycopg2',
+    'NAME': os.environ.get('NAME'),
+    'USER': os.environ.get('USER'),
+    'PASSWORD': os.environ.get('PASSWORD'),
+    'HOST': os.environ.get('HOST'),
+    'PORT': '5432',
+    }
+}
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = 'qi8da5)fd@k^3!mr5l85^)f+#$w%9jq2c&4rg)-s=w3)!dwnd4'
```

Comment:
What happened to not sharing this with anybody?

Reply:
- Ensure resolved with the setting of the environmental variables